wasm-runtime/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
```

Comment:
I find it really useful to add there something like `set -e`.

So if running this

```
./build.sh && RUST_BACKTRACE=1 cargo test -p polkadot-executor
```

with `wasm-runtime` being broken will fail with an error rather than proceeding to execute tests in `polkadot-executor` package.